pkg/util/hlc/hlc.go

```diff
@@ -27,6 +27,19 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/log"
 	"github.com/cockroachdb/cockroach/pkg/util/syncutil"
 	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
+	"github.com/pkg/errors"
+)
+
+const (
+	// The following constants are used if server.hlc.persist_future_wall_time
+	// is true
+
+	// persistFutureWallTimeInterval is the interval used for persisting the
+	// future wall time
+	persistFutureWallTimeInterval = 400 * time.Millisecond
+	// persistFutureWallTimeDelta is the delta used to compute the future wall
+	// time. This should be greater than persistFutureWallTimeInterval
+	persistFutureWallTimeDelta = int64(15 * time.Second)
```

Comment:
These constants will need to be tuned. Making them configurable through a cluster setting makes things a bit ugly